CONTRIBUTING.md

```diff
@@ -18,11 +18,11 @@ Please see https://git.k8s.io/community/CLA.md for more info.
 
 ## Contributing steps
 
-1. Submit an issue describing your proposed change to the repo in question.
-1. The [repo owners](OWNERS) will respond to your issue promptly.
-1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
-1. Fork the desired repo, develop and test your code changes.
-1. Submit a pull request.
+1. Submit an issue describing your proposed change to the repo in question. 
+2. The [repo owners](OWNERS) will respond to your issue promptly. 
+3. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above). 
+4. Fork the desired repo, develop and test your code changes. 
+5. Submit a pull request.
```

Comment:
The extra spaces would make the CI tests fails. We check it to not allow we merge docs with extra spaces